x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/processes/processes.tsx

```diff
@@ -46,10 +51,14 @@ export const Processes = () => {
   const ref = useRef<HTMLDivElement>(null);
   const { getDateRangeInTimestamp } = useDatePickerContext();
   const [urlState, setUrlState] = useAssetDetailsUrlState();
-  const { asset } = useAssetDetailsRenderPropsContext();
+  const { asset, renderMode } = useAssetDetailsRenderPropsContext();
   const { sourceId } = useSourceContext();
   const { request$ } = useRequestObservable();
   const { isActiveTab } = useTabSwitcherContext();
+  const { dataStreams, status: dataStreamsStatus } = useAssetEntitySummary({
+    entityType: 'host',
```

Comment:
There are some other places with `host` and `container`, perhaps we could replace them with `EntityType`. Wdyt?

```suggestion
    entityType: EntityType.HOST,
```